addon/components/image-magnifier.js

```diff
@@ -23,6 +23,25 @@ export default Component.extend({
       return this.get('previewerTarget') || `#${this.elementId}`;
     }
   }),
+  
+  lensWidth: computed('zoom', '_previewerWidth', function () {
+    return this.get('_previewerWidth') / this.get('zoom');
+  }),
+  
+  lensHeight: computed('zoom', '_previewerHeight', function () {
+    return  this.get('_previewerHeight') / this.get('zoom');
+  }),
+  
+  _previewerWidth: computed('previewerWidth', 'magnifierDimensions.width', {
+    get() {
+      return this.getWithDefault('previewerWidth', this.get('magnifierDimensions.width'));
+    }
+  }),
+  _previewerHeight: computed('previewerHeight', 'imgRatio', '_previewerWidth', {
+    get() {
+      return this.getWithDefault('previewerHeight', this.get('_previewerWidth')/this.get('imgRatio'));
+    }
```

Comment:
@selvagsz Made the changes, have used `setPreviewerDimensions` to compute default preview dimensions.